premake5.lua

```diff
@@ -4,7 +4,7 @@ workspace "OpenParrot"
 
 	flags { "StaticRuntime", "No64BitChecks" }
 
-	systemversion "10.0.16299.0"
+	systemversion "latest"
```

Comment:
This doesn't work, you need to use version numbers, otherwise vs2017 won't load the solution. (10.0.17763.0 is the current latest)

```suggestion
	systemversion "10.0.17763.0"
```

Reply:
Needs Premake alpha 13 or newer, https://github.com/premake/premake-core/releases (I didn't include it in the commit, and I updated the readme)

Reply:
Please remove this latest

Reply:
done